source/sync/go-to-production/optimize-sync-atlas-usage.txt

```diff
@@ -36,22 +36,42 @@ underlying data <changesets>` for each realm, similar to the
 uses this history to synchronize data between the backend and clients.
 App Services stores history in your synced Atlas cluster.
 
+Optimization Methods
+--------------------
+
+There are two methods for optimizing utility data synced in your cluster
```

Comment:
[nit] Curious if we want to start using the term "utility data". If so thats fine with me, but I think we historically have used "metadata"